cdap-docs/_common/_themes/cdap/static/cdap.css_t

```diff
@@ -214,17 +200,34 @@ div.body h4 { background-color: #EDEDED; margin: 30px 0px 10px 0px; }
 div.body h5 { background-color: #F2F2F2; margin: 30px 0px 10px 0px; }
 div.body h6 { background-color: #F2F2F2; }
 
+a {
+    color: #ff6600;
+    text-decoration: none;
+}
+
+a.current {
+    font-weight: bold;
+}
+
+a:hover {
+    color: #005B81;
+    text-decoration: underline;
+}
+
 a.headerlink {
-    color: #c60f0f;
+    color: #ff6600 !important;
```

Comment:
Its not a good pattern to use `!important` in css. This will prevent anybody to override this style using any other css selector in the future. If the styling is lacking specificity then we should create more specific rules to style the elements.